lib/rules/no-use-before-define.js

```diff
@@ -39,6 +39,14 @@ function parseOptions(options) {
     return { functions, classes, variables, typedefs };
 }
 
+/**
+ * @param {Scope} scope - a scope to check
+ * @returns {boolean} `true` if the scope is toplevel
+ */
+function isToplevelScope(scope) {
```

Comment:
Casing of the method...

```suggestion
function isTopLevelScope(scope) {
```